code-challenges.test.js

```diff
@@ -22,13 +35,94 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+const fibSeq = (fibSeqLen) => {
+  let newArray = []
+  let x = 1
+  let y = 1
+  let z = null
+  newArray.push(x)
+  newArray.push(y)
+  for (let i = 2; i < fibSeqLen; i++) {
+      z = x + y
+      x = y
+      y = z
+      newArray.push(z)
+  } return newArray
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
+//Pseudo Code: For Above
+// 1. Create a function named fibSeq.
+    // 1a. Has one parameter named fibSeqLen.
+// 2. Declare 4 new variables.
+    // 2a. First; newArray = []; Since output is an array.
+    // 2b. Second; x = 1, Third; y = 1
+    // 2c. Fourth; z = null; intentional absence of a value.
+    // 2d. Use built in method .push for both x and y into the newArray. This will give us the expected output for the first two elements.
+// 3. Create a for loop, starting at 2, ending at our parameter -since our parameter datatype is a number-, listing one by one.
+// 4. Within the for loop;
+    // 4a. z = x + y; x and y are defined above outside the for loop at the start.
+              // x and y will change depending on our value(s) or sum of 4b and 4c.
+    // 4b. x = y; stating x = y.
+              // This value will change depending on our value from 4c. 
+    // 4c. y = z; stating y = z
+              // This value will change depending on our sum from 4a.
+    // 4d. Use .push to push the new value of z into the array.
+// 5. return newArray outside of the for loop.
+
+// Pseudo code: for below refactored from above.
+// 1. Create a function named fibSeq. Has one parameter named fibSeqLen.
+// 2. Declare a new variable named newArr equal to an array of [1, 1]. These Fixed values are in place for our x and y.
+// 3. Create a for loop with build in methods for starting and ending for i. 
+    //3a. Starting newArr.length, and ending will be our parameter.
+    //3b. Execute the action of the for loop for i. 
+    //3c. Attach bracket notation to newArr[i]. Have it equal a arithmetic equation. newArr[i-1] + newArr[i-2]. 
```

Comment:
Love the refactor, along with new pseudo code to match it, great work. The only thing I would suggest would be lines 79-81, it is a little confusing on how you worded it there. Here's how I would write it, just as a reference:
- iterate using for loop from index 2 to index fibSeqLen
- each iteration makes current index value equal to the sum of last 2 index values

Keeps it nice and short, explains what the code is doing, and personally I would add each of those lines on top of lines 86 and 87, respectively, so that the pseudo code is sitting right above the line of code it is explaining. Overall great work on this, big win solving it!